chart/values.yaml

```diff
@@ -6,10 +6,10 @@ customCaCertUsed: false
 caCertSecretName: &caCertSecretName "cluster-ca-secret"
```

Comment:
Yaml anchors look pretty good, but in case of Helm the most suitable tool to reuse code is `_helpers.tpl` file